docs/cli-kit/ui-kit/contributing.md

```diff
@@ -0,0 +1,241 @@
+# Contributing to UI Kit
+
+## What is UI Kit
+
+UI Kit is a library that can be used to either ask for input from the user or render output to the terminal.
+By using UI Kit you're making sure that whatever you're rendering in the terminal will follow our design system.
```

Comment:
Do we have a design system? Should we say that "we ensure that your terminal experiences are consistent with the rest of the CLI?"

Reply:
We do have a design system :-) We should add more info about that linked from here cc @MeredithCastile

Reply:
I've rephrased this to not mention the design system for now.